maestro/src/agents/agent.py

```diff
@@ -17,22 +18,36 @@ def __init__(self, agent:dict) -> None:
             agent_name (str): The name of the agent.
         """
         # TODO: Review which attributes belong in base class vs subclasses
```

Comment:
Let's make sure to associate TODOs with issues. Want to avoid endless ToDOs or comments in code that are never done :)